src/sentinel.c

```diff
@@ -2256,11 +2256,13 @@ void sentinelFlushConfig(void) {
     if ((fd = open(server.configfile,O_RDONLY)) == -1) goto werr;
     if (fsync(fd) == -1) goto werr;
     if (close(fd) == EOF) goto werr;
-    return;
+    serverLog(LL_WARNING,"Save new sentinel configuration successfully on disk!");
+    return C_OK;
 
 werr:
     serverLog(LL_WARNING,"WARNING: Sentinel was not able to save the new configuration on disk!!!: %s", strerror(errno));
     if (fd != -1) close(fd);
+    return C_ERR;
```

Comment:
- Please avoid exclamation marks
- There is no need to write `WARNING:` (To verify my claim, I searched across the project for LL_WARNING to review convention)
- Consider rephrase to: `Failed to save Sentinel new configuration on disk`

Reply:
Addressed. Thanks